### PR TITLE
Read only

### PR DESCRIPTION
**Pull request by joshmoore**
Example of having a read-only user created for a given database. This is for use with read-only OMERO (cF. https://github.com/openmicroscopy/openmicroscopy/pull/5213 etc)

The permissions can only be set _after_ the database has been created. If this is to be part of this or another role, then either table creation will need to take place here or a second role will need to be created.

### CODE REVIEW

**Comment by manics** [comment]
Maybe a second role would be easier, especially if omero-read-only becomes widespread? Either a role which operates on existing databases, or if you prefer a postgres modification role specific to omero-read-only?

**Comment by manics** [comment]
The danger with removing the `idempotence` step is if in future someone makes a core change that isn't idempotent you get a cascading effect where other roles and playbooks that use this role also fail the idempotence check.

**Comment by manics** [comment]
Since this is effectively an example (there aren't any changes to the role) how about one of:
- move this to a development role
- move this to an example playbook, e.g. https://github.com/openmicroscopy/ansible-public-omero-example or a new omero-readonly playbook.

**Comment by joshmoore** [comment]
That all matches my concern(s) as well. I'm certainly working towards a `read-only` role, but I don't know if I'll make it in a timely fashion. Regardless, I'll look into one of the options, but this PR does have some valid tests, so I'll revert the idempotency test and get things passing then merge as "more tests"

File: playbook.yml [hunk 1]
```diff
@@ -10,3 +10,20 @@
         password: bob123
         databases: [publicdb]
         roles: "CREATEDB,NOSUPERUSER"
+      - user: charles
+        password: charles123
+        databases: [publicdb]
+        privs: "SELECT"
```
Comment (manics) [inline playbook.yml:16]: `privs` is currently unused

**Review by manics — COMMENTED** [review]
Otherwise looks fine

File: playbook.yml (hunk 1 above)
Comment (joshmoore) [reply playbook.yml:16]: Ah, that explains something. I'm inclined to get this in (without tag) and we can come back to "privs" in https://github.com/openmicroscopy/ansible-role-postgresql/issues/5

**Pull request merged**